pgb_utils/requirements.txt

```diff
@@ -1,15 +1,14 @@
-# apache_beam[gcp]
-apache_beam
-aplpy
-astropy==3.2.1
-astropy-healpix==0.6
-beautifulsoup4==4.8
+# aplpy
+astropy
+# astropy==3.2.1
+# astropy-healpix==0.6
+# beautifulsoup4==4.8
```

Comment:
Echo of zoom convo: Fix requirement versions and add a conda .yml file.

Reply:
✅